src/components/form/PowerObjectData.js

```diff
@@ -0,0 +1,35 @@
+const power = {
+
+
+  power_name: "",
+
+
+  power_description: {
+
+      // powers will have 2 frequencies, the second will be "None" by default, and can be changed by the user
+      // this allows for a 2nd frequency without it being required, if None, do not print on sheet, but keep track of in data
+
+      // likely best handled by a drop down list, options (in order of priority) are: At-Will, Cyclical, Battle-Based, Recharge, Daily, and Other
+      // i.e. if color coding a power that is Battle-Based and daily, color it as a Battle-Based power
+      power_frequency_1: "", // *****
```

Comment:
Should frequency and uses still be nested in power description? I was thinking any header content (name, frequencies, # uses) would be un-nested, but let me know if this changed